### PR TITLE
Delete Article/Event

### PR DESCRIPTION
**Pull request by copy4ever**

### CODE REVIEW

File: NFproject/Events/urls.py [hunk 1]
```diff
@@ -6,5 +6,6 @@
     path('', views.events_list, name='events_list'),
     path('add_event/', views.add_event , name='add_event'),
     path('update_event/<event_slug>', views.update_event, name="update_event"),
+    path('delete_event/<event_id>', views.delete_event, name= "delete_event"),
```
Comment (Octowl) [inline NFproject/Events/urls.py]: `<int:event_id>`

File: NFproject/Events/views.py [hunk 2]
```diff
@@ -33,6 +33,10 @@ def update_event(request, event_slug):
         }
         return render(request, "update_event.html", context)
 
+def delete_event(request, event_id):
+    Events.objects.filter(id = event_id).delete()
```
Comment (Octowl) [inline NFproject/Events/views.py]: use `objects.get`

File: NFproject/Events/views.py [hunk 3]
```diff
@@ -33,6 +33,10 @@ def update_event(request, event_slug):
...
+    return redirect(events_list)
```
Comment (Octowl) [inline NFproject/Events/views.py]: `'events_list'`

File: NFproject/articles/urls.py [hunk 4]
```diff
@@ -6,5 +6,6 @@
     path('', views.article_list, name='article_list'),
     path('add_article/', views.add_article , name='add_article'),
     path('update_article/<article_slug>', views.update_article, name="update_article"),
+    path('delete_article/<article_id>', views.delete_article, name= "delete_article"),
```
Comment (Octowl) [inline NFproject/articles/urls.py]: `int`

File: NFproject/articles/views.py [hunk 5]
```diff
@@ -31,6 +31,10 @@ def update_article(request, article_slug):
         }
         return render(request, "update_article.html", context)
 
+def delete_article(request, article_id):
+    Article.objects.filter(id = article_id).delete()
```
Comment (Octowl) [inline NFproject/articles/views.py]: `.get`

File: NFproject/articles/views.py [hunk 6]
```diff
@@ -31,6 +31,10 @@ def update_article(request, article_slug):
...
+    return redirect(article_list)
```
Comment (Octowl) [inline NFproject/articles/views.py]: `'article_list'`

**Comment by copy4ever** [comment]
Done

**Comment by Octowl** [comment]
closes #68

**Pull request merged**